fhir-server/pom.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>fhir2</artifactId>
+		<groupId>org.openmrs.module</groupId>
+		<version>1.4.0-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>fhir-server</artifactId>
+	<name>FHIR2-server Rest Client </name>
+
+	<dependencies>
```

Comment:
For this module we should have absolutely no dependencies on other parts of the FHIR2 module except maybe the test-data if you want to reuse those definitions. We probably don't need the dependency on OpenMRS either. We should have an explicit dependency on the hapi-fhir-client project.